LICENSE

```diff
@@ -1,7 +1,5 @@
 MIT License
```

Comment:
Should we switch to `Apache License v2.0 with LLVM Exceptions`  as per `LICENSE.APACHE_LLVM`?